src/types/entities.ts

```diff
@@ -262,11 +262,72 @@ export const UserSchema = z.object({
     email: z.string(),
 })
 /**
- * Represents a user in Todoist.
+ * Represents a user in Todoist (simplified for collaborators).
  * @see https://todoist.com/api/v1/docs#tag/User
  */
 export type User = z.infer<typeof UserSchema>
 
+export const CurrentUserSchema = z
+    .object({
+        id: z.string(),
+        email: z.string(),
+        full_name: z.string(),
+        avatar_big: z.string().nullable(),
+        avatar_medium: z.string().nullable(),
+        avatar_s640: z.string().nullable(),
+        avatar_small: z.string().nullable(),
+        business_account_id: z.string().nullable(),
+        is_premium: z.boolean(),
+        date_format: z.number().int(),
+        time_format: z.number().int(),
+        weekly_goal: z.number().int(),
+        daily_goal: z.number().int(),
+        completed_count: z.number().int(),
+        completed_today: z.number().int(),
+        karma: z.number(),
+        karma_trend: z.string(),
+        lang: z.string(),
+        next_week: z.number().int(),
+        start_day: z.number().int(),
+        start_page: z.string(),
+        timezone: z.string(),
+        inbox_project_id: z.string(),
+        days_off: z.array(z.number().int()),
+        weekend_start_day: z.number().int(),
```

Comment:
These should be in camalCase, the conversion from snake > camel is handled by our axios middleware.